modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/SiteLevelWalkthrough.testcase

```diff
@@ -0,0 +1,61 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property osgi.modules.includes = "frontend-js-walkthrough-sample-web";
```

Comment:
The walkthrough sample portlet is not used at all in the test. This can be removed since we're only using site level JSON to create the walkthrough components.